app-guides/supercronic.html.md

```diff
@@ -82,10 +82,10 @@ Now that we've added `supercronic` to our `Dockerfile`, put the `crontab` at the
 $ fly deploy
 ```
 
-Then we'll need to scale the processes so that we only run one virtual machine container with the `cron` process. Be sure to change `web` to whatever number you had before.
+Then we'll need to scale the processes so that we only run one virtual machine container with the `cron` process.
```

Comment:
I'm not sure we should delete the part about the `web` process since this guide includes creating it that process (I assume it replaces the default `app` process present in a basic app.